Libs/Visualization/VTK/Widgets/ctkVTKScalarsToColorsWidget.cpp

```diff
@@ -549,6 +562,21 @@ void ctkVTKScalarsToColorsWidget::setYRange(double min, double max)
     }
 }
 
+// ----------------------------------------------------------------------------
+void ctkVTKScalarsToColorsWidget::resetRange()
+{
+  Q_D(ctkVTKScalarsToColorsWidget);
+  vtkAxis* xAxis = d->CurrentControlPointsItem ?
+    d->CurrentControlPointsItem->GetXAxis() : d->View->chart()->GetAxis(vtkAxis::BOTTOM);
+  vtkAxis* yAxis = d->CurrentControlPointsItem ?
+    d->CurrentControlPointsItem->GetYAxis() : d->View->chart()->GetAxis(vtkAxis::LEFT);
+  Q_ASSERT(xAxis);
```

Comment:
Instead (or maybe in addition) to the assert, we could call set?Range if ?axis is not NULL.

Reply:
I hate Q_ASSERT but added it because it's all over this class. You're right, I'll just add if's instead.